fastlane/metadata/steam/czech/achievements.vdf

```diff
@@ -0,0 +1,109 @@
+		"NEW_ACHIEVEMENT_20_12_NAME"	"Výzkumník"
+		"NEW_ACHIEVEMENT_20_12_DESC"	"Vyzkoumej všechno."
+		"NEW_ACHIEVEMENT_20_13_NAME"	"Měňavec"
+		"NEW_ACHIEVEMENT_20_13_DESC"	"Odemkni každého mech ve hře a transformuj se do něj."
+		"NEW_ACHIEVEMENT_20_14_NAME"	"Přetížení"
+		"NEW_ACHIEVEMENT_20_14_DESC"	"Zasáhni nepřítele pokrytého vodou energetickou zbraní."
+		"NEW_ACHIEVEMENT_20_15_NAME"	"Odražení"
+		"NEW_ACHIEVEMENT_20_15_DESC"	"Znič jednotku její vlastní odraženou kulkou."
+		"NEW_ACHIEVEMENT_20_17_NAME"	"Distributor v zácviku"
+		"NEW_ACHIEVEMENT_20_17_DESC"	"Vynalezni Směrovač."
+		"NEW_ACHIEVEMENT_20_18_NAME"	"Stvořitel"
+		"NEW_ACHIEVEMENT_20_18_DESC"	"Umísti 10.000 bloků."
+		"NEW_ACHIEVEMENT_20_19_NAME"	"Viktor čistič"
+		"NEW_ACHIEVEMENT_20_19_DESC"	"Znič 1.000 nepřítelských bloků."
+		"NEW_ACHIEVEMENT_20_20_NAME"	"Velkolepá chyba"
+		"NEW_ACHIEVEMENT_20_20_DESC"	"Způsob přetížení Thoriového reaktoru, následované explozí."
+		"NEW_ACHIEVEMENT_20_21_NAME"	"Kartograf"
+		"NEW_ACHIEVEMENT_20_21_DESC"	"Vytvoř 10 nových map."
+		"NEW_ACHIEVEMENT_20_22_NAME"	"Sosač"
+		"NEW_ACHIEVEMENT_20_22_DESC"	"Stáhni mapu z dílny."
+		"NEW_ACHIEVEMENT_20_23_NAME"	"Tvůrce"
+		"NEW_ACHIEVEMENT_20_23_DESC"	"Publikuj mapu do dílny."
+		"NEW_ACHIEVEMENT_20_24_NAME"	"Řezník"
+		"NEW_ACHIEVEMENT_20_24_DESC"	"Poraž Záporáka."
+		"NEW_ACHIEVEMENT_20_25_NAME"	"Průzkumník"
+		"NEW_ACHIEVEMENT_20_25_DESC"	"Odemkni všechny zóny v kampani."
+		"NEW_ACHIEVEMENT_20_26_NAME"	"Perfektionalista"
+		"NEW_ACHIEVEMENT_20_26_DESC"	"Dosáhni požadavku pro vlastní výbavu ve všech zónách v kampani."
+		"NEW_ACHIEVEMENT_20_29_NAME"	"Materiály II"
+		"NEW_ACHIEVEMENT_20_29_DESC"	"Odemkni Thorium."
+		"NEW_ACHIEVEMENT_20_31_NAME"	"Materiály I"
+		"NEW_ACHIEVEMENT_20_31_DESC"	"Odemkni Titan."
+		"NEW_ACHIEVEMENT_21_0_NAME"	"Kamikaze"
+		"NEW_ACHIEVEMENT_21_0_DESC"	"Naplň svého mecha výbušným materiálem a zemři v explozi ."
+		"NEW_ACHIEVEMENT_21_1_NAME"	"A tak to začíná"
+		"NEW_ACHIEVEMENT_21_1_DESC"	"Vytvoř továrnu na mecha Dýka."
+		"NEW_ACHIEVEMENT_21_2_NAME"	"Přímý útok"
+		"NEW_ACHIEVEMENT_21_2_DESC"	"Vyšli příkaz k útoku z řídícího střediska."
+		"NEW_ACHIEVEMENT_21_3_NAME"	"Roj"
+		"NEW_ACHIEVEMENT_21_3_DESC"	"Měj v jeden ukamžil 100 aktivních jednotek."
+		"NEW_ACHIEVEMENT_21_4_NAME"	"Stádo"
+		"NEW_ACHIEVEMENT_21_4_DESC"	"Měj v jeden okamžik 10 aktivních dronů Fantóm ."
+		"NEW_ACHIEVEMENT_21_5_NAME"	"Proměnlivá armáda"
+		"NEW_ACHIEVEMENT_21_5_DESC"	"Měj v jeden okamžik 50 aktivních jednotek Slídič."
+		"NEW_ACHIEVEMENT_21_6_NAME"	"Legie"
+		"NEW_ACHIEVEMENT_21_6_DESC"	"Postav celkem 1.000 jednotek."
+		"NEW_ACHIEVEMENT_21_7_NAME"	"Super"
+		"NEW_ACHIEVEMENT_21_7_DESC"	"Dosáhni hodnocení S v jakékoliv zóně."
+		"NEW_ACHIEVEMENT_21_8_NAME"	"Super Super"
+		"NEW_ACHIEVEMENT_21_8_DESC"	"Dosáhni hodnocení SS v jakékoliv zóně."
+		"NEW_ACHIEVEMENT_21_9_NAME"	"Měl jsi líp poslouchat"
+		"NEW_ACHIEVEMENT_21_9_DESC"	"Zemři v zóně pro vylíhnutí nepřítelských jednotek."
```

Comment:
nepřítelských => nepřátelských

Reply:
jo, tohle je můj častý překlep, opravovil jsme to teda hromadně na 5 místech :) díky